app/routes.py

```diff
@@ -1 +1,245 @@
-from flask import Blueprint
\ No newline at end of file
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from flask import Blueprint, jsonify, make_response, request, abort
```

Comment:
👍🏾 These imports help a great deal when it comes to our request & response cycle